README.md

```diff
@@ -33,15 +33,15 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. What is the difference between an adaptive website and a fully responsive website?
-
+    Adaptive relates to using server logic to tell what size screen is being used, responisve uses media queires to detect the device being used.
 2. Describe what it means to be mobile first vs desktop first.
-
+    Mobile first means building from the small end of screen size (mobile) and up, also using min-width instead of max width. Desktop is opposite, going larger down, using max-width.
 3. What does `font-size: 62.5%` in the `html` tag do for us when using `rem` units?
-
+    It makes the rem units factors of 10, because the default text size is 16 px. Doing this makes the default 10px.
 4. How would you describe preprocessing to someone new to CSS?
-
+    It's a version of CSS that is smarter and better and has more options. It takes what you give it and translates it to regular CSS.
```

Comment:
Very close with the 'translates' answer. The preprocessor compiles your code; so, as you're working on it, you can use a more streamline format (readable and easy to apply), and then the CSS is compiled to its machine-readable format. Good work